helm_deploy/apply-for-legal-aid/values-production.yaml

```diff
@@ -54,6 +54,14 @@ ingress:
     - "apply-for-legal-aid.service.justice.gov.uk"
   secretName: "apply-for-legal-aid-tls-secret"
 
+publicIngress:
+  enabled: true
+  className: modsec
```

Comment:
The PR description says
> Note that this also disables modsec on the ingress for /ping and /status.

But this seems to say it _is_ implemented :/

Reply:
@colinbruce thanks for the review. I have left `modsec` as the `className` so that it is "enablable" but staging and production both have an annotation, as below, that disables it.
```
nginx.ingress.kubernetes.io/enable-modsecurity: "false"
```

I was not sure whether we wanted to use modsec or not for this ingress 🤷 so left it enableable in case. What do you think? Should we completely remove it? `#ping` is a very basic action so do not see how modesec adds anything.